src/Infiltrator.jl

```diff
@@ -159,57 +82,102 @@ const TEST_TERMINAL_REF = Ref{Any}(nothing)
 const TEST_REPL_REF = Ref{Any}(nothing)
 const TEST_NOSTACK = Ref{Any}(false)
 
-const EXITING = Ref{Bool}(false)
-const DISABLED = Set()
+mutable struct ScratchPad
+  store::Module
+  exiting::Bool
+  disabled::Set
+end
+function Base.show(io::IO, s::ScratchPad)
+  n = length(get_scratch_pad_names(s))
+  print(io, "Infiltrator scratch pad with $(n) name$(n == 1 ? "" : "s")")
+end
+function Base.getproperty(sp::ScratchPad, s::Symbol)
```

Comment:
We can use `getfield` or other helper functions to avoid name collisions? the `=== :store` seems a bit fragile.